lib/ReactViews/Workbench/Controls/ViewingControls.jsx

```diff
@@ -111,7 +111,16 @@ const ViewingControls = observer(
 
     zoomTo() {
       const viewer = this.props.viewState.terria.currentViewer;
-      viewer.zoomTo(this.props.item);
+      const item = this.props.item;
+      let zoomToView = item;
+      if (
+        item.rectangle !== undefined &&
+        item.rectangle.east - item.rectangle.west >= 360
```

Comment:
The only question mark I have about this is whether it will work for things in the Pacific that cross the anti-meridian...

Reply:
I think if it does get to beyond 360 degrees from east to west there would still be something wrong with the bounding box, given that the only scenario this would break is if the east >= 180 and west <= -180, in which case the east should be the west value and vice versa and it seems like a weird bounding box anyway? 🤔 Though correct me if I'm wrong but I don't think it would break pacific map (as I imagine the behaviour in master doesn't break it either)

Reply:
I'm thinking even in the worst case scenario where a bounding box has `east: 179.9` and `west`-179.9`, which is reasonably likely given the location of pacific islands 😂 , it still would zoom to the desired location

Reply:
Given that I can't find a good example and it's an edge case let's just role with what you've done :)